www/src/pages/foundations/typography.tsx

```diff
@@ -42,356 +101,233 @@ const measuredTypeProps = {
-                  </a>{' '}
-                  in a sentence.
-                </p>
-              </td>
-              <td>
-                <small>
-                  For <code>.muted-link</code> links inside a <code>p</code> or
-                  with the <code>.inline-link</code> class name.
-                </small>
-              </td>
-            </tr>
-          </tbody>
-        </table>
-
-        <table className="w-100 table pgn-doc__status-table">
-          <tbody>
-            <tr>
-              <th colSpan={2}>
-                <h2>Decoration and Emphasis</h2>
-              </th>
-            </tr>
-            <tr>
-              <th>Style</th>
-              <th>CSS Class</th>
-            </tr>
-            <tr>
-              <td>
-                <p className="text-lowercase">Lowercase text.</p>
-              </td>
-              <td>
-                <code>.text-lowercase</code>
-              </td>
-            </tr>
-            <tr>
-              <td>
-                <p className="text-uppercase">uppercase text.</p>
-              </td>
-              <td>
-                <code>.text-uppercase</code>
-              </td>
-            </tr>
-            <tr>
-              <td>
-                <p className="text-capitalize">capitalize text.</p>
-              </td>
-              <td>
-                <code>.text-capitalize</code>
-              </td>
-            </tr>
-            <tr>
-              <td>
-                <p className="text-decoration-none">No decorations.</p>
-              </td>
-              <td>
-                <code>.text-decoration-none</code>
-              </td>
-            </tr>
+                ),
+              },
+              {
+                Header: 'Mobile',
+                Cell: ({ row }: DataTableTypes) => (
+                  <div className="mobile-type">
+                    <MeasuredItem {...measuredTypeProps}>
+                      <p className={`m-0 display-${row.original.size}`}>
+                        Display {row.original.size}
+                      </p>
+                    </MeasuredItem>
+                  </div>
+                ),
+              },
+              {
+                Header: 'CSS Class',
+                Cell: ({ row }: DataTableTypes) => (
+                  <code style={{ fontSize: '16px' }}>
+                    .display-{row.original.size}
+                  </code>
+                ),
+              },
+            ]}
+          >
+            <DataTable.Table />
+          </DataTable>
+        </div>
 
-            <tr>
-              <td>
-                <p className="text-italic">Italic text.</p>
-              </td>
-              <td>
-                <code>.text-italic</code>
-              </td>
-            </tr>
+        <h2 className="mb-2">Links</h2>
+        <div className="mb-4">
+          <DataTable
+            itemCount={3}
+            data={linksClassesAndDescriptions.map(({ name, text }) => ({ name, text }))}
```

Comment:
no need to do `.map` here, data={linksClassesAndDescriptions} should work

Reply:
Thanks, simplified